storage-controllers/s3.controller.php

```diff
@@ -16,7 +16,7 @@ function connect(){
 		require_once(ROOT.DS.'storage-controllers'.DS.'s3'.DS.'aws-autoloader.php');
 		$this->s3 = new Aws\S3\S3Client([
 			'version' => 'latest',
-			'region'  => 'us-east-1',
+			'region'  => S3_REGION,
```

Comment:
can you makie it backwards compatible please so existing configs won't brick it.

eg `(is_defined('S3_REGION') && S3_REGION ?S3_REGION:'us-east-1')`